README.md

```diff
@@ -83,5 +101,45 @@ date = 1970-01-01
 author = "John Smith"
 ```
 
-## Original
-This template is based on the Hugo template https://git.habd.as/comfusion/after-dark
+### Code Snippets
+
+Syntax Highlighting:
+
+```toml
+[markdown]
+# Whether to do syntax highlighting
+# Theme can be customized by setting the `highlight_theme` variable to a theme supported by Zola
+highlight_code = true
+
+highlight_theme = "one-dark"
+```
+
+Enhanced Codeblocks (Clipboard Support and Language Tags)
+
+```toml
+[extra]
+codeblock = true
```

Comment:
Can you add the same line as for latex/search bar about putting it in config.toml?